src/pages/Login/Login.js

```diff
@@ -1,7 +1,93 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './Login.scss';
 
 const Login = () => {
-  return <div>Login</div>;
+  const [userInputs, setUserInputs] = useState({
+    userId: '',
+    userPassword: '',
+  });
+
+  const navigate = useNavigate();
+
+  const { userId, userPassword } = userInputs;
+
+  const isValid = userId.includes('@') && userPassword.length >= 5;
+
+  const handleLoginBtn = () => {
+    if (!isValid) {
+      alert('이메일 혹은 비밀번호를 확인해주세요');
+
+      return;
+    }
+
+    fetch('http://10.58.52.136:3000/users/signin', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        email: userId,
+        password: userPassword,
+      }),
+    })
+      .then(res => res.json())
+      .then(data => {
+        if (data.accessToken) {
+          localStorage.setItem('token', data.accessToken);
+          navigate('/main');
+        } else if (data.message === 'INVALID_USER') {
+          alert('이메일 혹은 비밀번호를 확인 해 주세요');
+        }
+      });
+  };
+
+  const handleUserInputs = e => {
+    const { name, value } = e.target;
+    setUserInputs({
+      ...userInputs,
+      [name]: value,
+    });
```

Comment:
```suggestion
    setUserInputs(prev => ({
      ...prev,
      [name]: value,
    }));
```
이 부분 함수형 업데이트로 구현해주세요~